randovania/games/am2r/logic_database/Genetics Laboratory.txt

```diff
@@ -706,7 +706,10 @@ Extra - minimap_data: [{'x': 7, 'y': 20}, {'x': 8, 'y': 20}, {'x': 9, 'y': 20}]
                       Missiles ≥ 30 or Super Missiles ≥ 10
                       Any of the following:
                           # Defend against blobs
-                          Plasma Beam or Combat (Expert)
+                          Combat (Expert)
+                          Plasma Beam and Spazer Beam
+                          Plasma Beam and Combat (Beginner)
+                          Spazer Beam and Wave Beam and Combat (Intermediate)
```

Comment:
i wonder whether we should do this instead:
- plasma AND spazer AND wave
- plasma AND spazer (beginner)
- plasma AND wave (beginner)
- plasma (intermediate)
- spazer AND wave (intermediate)

Reply:
Does `wave + plasma` really help = `spazer + plasma`? I'd think the extra wide beam is much more of an advantage.

Reply:
both help
wave makes things wider and does more damage, but you have a tad more shoot delay
spazer makes things only a bit wider, doesnt deal as much damage, but you can spam more.